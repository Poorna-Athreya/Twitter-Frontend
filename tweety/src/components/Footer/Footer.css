footer{
    width: 100%;
    height: 8vh;
    background-color: #0080FF;
    color: white;
    display: flex;
    align-items: center;
    padding-left: 3%;
}