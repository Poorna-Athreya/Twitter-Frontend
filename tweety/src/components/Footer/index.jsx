import React from 'react';
import './Footer.css';

function Footer() {
  return (
    <footer className="footer">
      <p>copyright@2022</p>
    </footer>
  );
}
export default Footer;
