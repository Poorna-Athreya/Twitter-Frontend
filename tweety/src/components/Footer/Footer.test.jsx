import { render, screen } from '@testing-library/react';
import React from 'react';
import Footer from './index';

describe('Footer', () => {
  it('should take a snapshot for Footer', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
  it('should have the required elements when Footer is rendered', () => {
    render(<Footer />);
    expect(screen.queryByTestId('footerText')).toBeTruthy();
    expect(screen.getByText('copyright@2022')).toBeTruthy();
  });
});
