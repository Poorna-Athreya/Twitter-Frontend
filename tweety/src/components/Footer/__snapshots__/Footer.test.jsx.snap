// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should take a snapshot for Footer 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p
      data-testid="footerText"
    >
      copyright@2022
    </p>
  </footer>
</DocumentFragment>
`;
